src/app.js

```diff
@@ -2,4 +2,163 @@ const express = require('express');
 
 const app = express();
 
+app.use(express.json());
+
+const {
+  sayHello,
+  uppercase,
+  lowercase,
+  firstCharacter,
+  firstCharacters,
+} = require('./lib/strings');
+
+const { add, multiply, subtract, divide, remainder } = require('./lib/numbers');
+
+const {
+  getNthElement,
+  arrayToCSVString,
+  addToArray,
+  elementsStartingWithAVowel,
+  removeNthElement2,
+} = require('./lib/arrays');
+
+const { negate, truthiness, isOdd, startsWith } = require('./lib/booleans');
+
+app.get('/strings/hello/:string', (req, res) => {
+  const { string } = req.params;
+  res.json({ result: sayHello(string) });
+});
+
+app.get('/strings/upper/:string', (req, res) => {
+  const { string } = req.params;
+  res.json({ result: uppercase(string) });
+});
+
+app.get('/strings/lower/:string', (req, res) => {
+  res.json({ result: lowercase(req.params.string) });
+});
+
+app.get('/strings/first-characters/:string', (req, res) => {
+  if (!req.query.length) {
+    res.json({ result: firstCharacter(req.params.string) });
+  } else {
```

Comment:
You could delete the else statement here and just have 
```
    res.json({ result: firstCharacters(req.params.string, req.query.length) });
````
after the if statement :)